docker-compose.yml

```diff
@@ -19,3 +19,17 @@ services:
       tty: true
       volumes:
         - '.:/sdk_codegen'
+
+    typescript:
+      environment:
+        - LOOKERSDK_BASE_URL=${LOOKERSDK_BASE_URL}
```

Comment:
You can add `LOOKERSDK_INI=/path/to/looker.ini` here if you want the test to using a different INI file

Reply:
Oh this is like this because I'm not sure if there is a way to set env variables for all containers.